README.md

```diff
@@ -1,2 +1,173 @@
-# Math-Magician
-"Math magicians" is a website for all fans of mathematics. 
+<a name="readme-top"></a>
+
+<div align="center">
+
+ <!-- LOGO -->
+
+  <img src="./src/assets/AYO&apos;S LOGO.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+<!-- MAIN HEADING -->
+
+  <h3><b>MATH MAGICIAN</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- INTRO -->
+# 📖Math Magician <a name="about-project"></a>
+
+> "Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:Make simple calculations & Read a random math-related quote.
+
+## 🛠 Built With <a name="built-with"></a>
+1. React
+2. CSS
+3. JavaScript
+4. Webpack
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React</a></li>
+    <li><a href="https://www.w3.org/">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+    <li><a href="https://webpack.js.org/">Webpack</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> - Perform simple and complex mathematical calculations
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+LIVE DEMO
+
+> Visit [live]() page of my Profile
```

Comment:
- [ ] You have added a broken link to your readme file, it is considered unprofessional to leave broken links in readmes. Seeing that your live demo is not available you can remove this section and display It when the deployed version is available or you can provide the appropriate link for user to view 👍